python/rainbow/simulators/prox_rigid_bodies/types.py

```diff
@@ -332,6 +332,7 @@ def __init__(self):
         """
         Create a default empty engine instance.
         """
+        self.body_type = 'rigid_body'
```

Comment:
Perhaps add a comment about what this field is used for?

The naming is a little strange. An engine has a body_type that is rigid_body?

Perhaps this is more a simulator_type that is "rigid bodies"?

I guess this is used elsewhere in the framework to abstract over rigid and soft body simulators. To make code sharing easier?